lib/dry/rails/container.rb

```diff
@@ -59,15 +59,13 @@ class Container < System::Container
       config.auto_registrar = Rails::AutoRegistrars::App
 
       class << self
-        # Return if a given component was booted
+        # Return if a given component was started
         #
         # @return [Boolean]
         #
         # @api private
-        #
-        # TODO: this should be moved to dry-system
-        def booted?(name)
-          booter.booted.map(&:name).include?(name)
+        def started?(name)
+          providers[name].started?
```

Comment:
@solnic Oh, this turned out to be much nicer than the previous version! Nice to see :)